Assets/Scripts/Game/UserInterfaceWindows/DaggerfallTravelPopUp.cs

```diff
@@ -325,6 +325,8 @@ private void performFastTravel()
         {
             RaiseOnPreFastTravelEvent();
 
+            DeductFastTravelGold();
```

Comment:
I would rather you moved this above "RaiseOnPreFastTravelEvent". 

Before this change, the player gold was removed before any code that reacted to this event. This means code might have been assuming that the gold was already taken care of, and change gold amount.

Reply:
Otherwise, all is as expected

Reply:
@KABoissonneault Done